test/runtests.jl

```diff
@@ -2859,5 +2859,18 @@ else  # run HiGHS tests
             @test length(r["Messages"]["errors"]) > 0
             @test length(r["Messages"]["warnings"]) > 0
         end
+
+        @testset "Storage Duration" begin
```

Comment:
@atpham88 could you add a test for either ColdThermalStorage or HotThermalStorage as well? I think the test will be helpful since it's not obvious (at least to me) how to calculate the "duration" of these from the outputs

Reply:
@adfarth tests for ColdThermalStorage or HotThermalStorage won't be necessary now since min/max_duration_storage are not options for them.